experiments/arctic_simulation.jl

```diff
@@ -63,12 +63,11 @@ using ClimaSeaIce.SeaIceMomentumEquations
 using ClimaSeaIce.Rheologies
 
 # Remember to pass the SSS as a bottom bc to the sea ice!
-SSS = view(ocean.model.tracers.S, :, :, grid.Nz)
+SSS = view(ocean.model.tracers.S.data, :, :, grid.Nz)
```

Comment:
why you need `.data` here?

Reply:
because otherwise the field can be indexed only with `grid.Nz`, i.e. we get a windowedfield with indices `(:, :, grid.Nz)`. This is something we need to fix in `ClimaSeaIce` probably